rubix-common/src/main/java/com/qubole/rubix/common/utils/DataGen.java

```diff
@@ -35,28 +35,38 @@ private DataGen()
   {
   }
 
-  public static String generateContent(int jump)
+  public static String generateContent(int jump, int inputSize)
```

Comment:
Change the variable name to reflect the iteration. Currently it feels the file size is the input size which is not the case